app/models/authenticator/amazon.rb

```diff
@@ -115,15 +116,11 @@ def verify_credentials(access_key_id, secret_access_key)
     end
 
     def aws_connect(access_key_id, secret_access_key, service = :IAM)
-      require 'aws-sdk-v1'
-
-      AWS.const_get(service).new(
-        :access_key_id     => access_key_id,
-        :secret_access_key => secret_access_key,
-
-        :logger            => $aws_log,
-        :log_level         => :debug,
-        :log_formatter     => AWS::Core::LogFormatter.new(AWS::Core::LogFormatter.default.pattern.chomp),
+      ManageIQ::Providers::Amazon::CloudManager.raw_connect_v2(
```

Comment:
not sure on this one, because it actually introduces coupling to the CloudManager. 
OTH looking at the actual code it makes sense to have those many options at one place: https://github.com/ManageIQ/manageiq/blob/b96cab9babd39c46bac552603445fc81301cf0d2/app/models/manageiq/providers/amazon/cloud_manager.rb#L75-L83